src/renderer/opengl/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ target_link_libraries(
 )
 
 if(NOT BUILD_SHARED_LIBS)
-    target_link_libraries(${NOX_LIB_NAME} PRIVATE ${NOX_OPENGL_LIB_NAME})
+    target_link_libraries(${NOX_IMPL_LIB_NAME} PRIVATE ${NOX_OPENGL_LIB_NAME})
```

Comment:
not really intentionally commited line 64 as we discussed nox_impl dependency on nox_opengl is going to be abandoned, however it fixed compilation issue on linux and I'm pretty sure it hasn't changed anything on windows

Reply:
not needed, should be `NOX_LIB_NAME`

Reply:
yep right now it compiles